enterprise/server/backends/distributed/distributed.go

```diff
@@ -97,6 +97,9 @@ func Register(env environment.Env) error {
 	if *listenAddr == "" {
 		return nil
 	}
+	if env.GetCache() == nil {
+		return status.FailedPreconditionErrorf("Distributed Cache requires a base cache; but one was not configured; please also enable a disk/pebble/memory cache")
```

Comment:
Can we make all of these consistent?
https://github.com/buildbuddy-io/buildbuddy/blob/master/enterprise/server/backends/redis_cache/redis_cache.go#L58
https://github.com/buildbuddy-io/buildbuddy/blob/master/enterprise/server/backends/memcache/memcache.go#L53